.github/workflows/sonar-project.properties.yml

```
@@ -0,0 +1,221 @@
+        push: true
+        tags: |
+          ${{ env.DOCKER_MIGRATION_IMAGE_NAME }}:${{ github.run_number }}
+          ${{ env.DOCKER_MIGRATION_IMAGE_NAME }}:latest
+        cache-from: type=registry,ref=${{ env.DOCKER_MIGRATION_IMAGE_NAME }}:latest
+        cache-to: type=inline
+
+  SonarQube-scan-and-quality:
+    runs-on: ubuntu-latest
+    steps:
+    - name: SonarQube scan
+      uses: actions/checkout@v2
```

actionlint (1.7.4)

the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)